packages/common/src/hooks/useAudioRewards.ts

```diff
@@ -49,7 +52,10 @@ export const useFormattedProgressLabel = ({
         formatNumberCommas(challenge?.max_steps?.toString() ?? '')
       )
     }
-  } else if (challenge?.challenge_type === 'aggregate') {
+  } else if (
+    challenge?.challenge_type === 'aggregate' &&
+    challenge?.max_steps !== null
```

Comment:
i know this has been discussed but can you remind - why do we need explicit null check instead of `!!challenge?.max_steps`?

Reply:
Both are actually fine in this case. I just went for very explicit since the value can explicitly be `null` or a number.